sample/helpers.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+
+def strip_prop(text, prop):
+    return re.sub(prop+':', "", text, count=1)
+
+
+def res_prop_list(res, prop, pageids):
+    return res['query']['pages'][pageids][prop]
+
+
+def beautiful(res, prop, strip_chars):
+    ret = {}
+    for pageid, page_content in res['query']['pages'].items():
+        if prop not in page_content:
+            continue
+        ret[pageid] = [strip_prop(content['title'], strip_chars) for content in page_content[prop]]
```

Comment:
@ayuhsya I meant `[strip_prop(entry['title'], strip_chars) for entry in page_content[prop]]`